src/vs/base/common/paths.ts

```diff
@@ -8,6 +8,7 @@ import { isLinux, isWindows } from 'vs/base/common/platform';
 import { fill } from 'vs/base/common/arrays';
 import { rtrim } from 'vs/base/common/strings';
 import { CharCode } from 'vs/base/common/charCode';
+import { endsWith } from 'vs/base/common/strings';
```

Comment:
it is used actually:
https://github.com/hun1ahpu/vscode/blob/896c9850b0394723aab783e39947613fa6597c59/src/vs/base/common/paths.ts#L425